styles/burger.css

```diff
@@ -0,0 +1,105 @@
+/* Burger menu*/
+
+@media screen and (max-width: 992px) {
```

Comment:
No need to declare the styles below as part of the media query because these are the basic styles and what you declared below:
```
@media (min-width: 992px) {
  .burger {
    display: none;
  }
}
```
will prevent them from being used on screens wider than 991px.